core/execution/future/market.go

```diff
@@ -3770,6 +3791,12 @@ func (m *Market) settlementDataPerp(ctx context.Context, settlementData *num.Num
 		return
 	}
 
+	for _, t := range transfers {
+		if t.Transfer() != nil && t.Transfer().Type == types.TransferTypeMTMWin {
```

Comment:
The types from the perpetual funding payments will only ever be `TransferTypePerpFundingWin` or `TransferTypePerpFundingLoss` here so I don't know if we necessarily need this here.

Reply:
so I imagine we need to check for TransferTypePerpFundingWin here too?

Reply:
Yea its all exactly the same at the MTM transfers so it'll work if you change the type to the perp-win one. My reading of the spec makes it seems like we do want these transfers included? But might want to check with research?

Reply:
@cdummett will confirm but for now adding wins and losses. (for some reason I forgot to add the mtm losses in the first place)